### DISCUSSION TITLE
Package Update on TF framework

### DISCUSSION THREAD

**Discussion by AlvinAi96** [PaddlePaddle/PaddleSpatial#196] [BaiduKDDCup2022] [answered]
Dear Host @ramber1836 ,

Would you like to update the TensorFlow version from 2.7 to 2.9.1?

Thanks.

Alvin

**Comment by ramber1836** [comment, marked as answer]
I don't think it is wise to upgrade TensorFlow.

**Reply by AlvinAi96** [reply]
Got it, thanks for replying. Lu.
